app/controllers/api/stateless/braintree/subscriptions_controller.rb

```diff
@@ -0,0 +1,30 @@
+module Api
+  module Stateless
+    module Braintree
+      class SubscriptionsController < StatelessController
+        before_filter :authenticate_request!
+
+        def index
+          @subscriptions = PaymentHelper::Braintree.subscriptions_for_member(@current_member)
+        end
+
+        def destroy
+          @subscription = PaymentHelper::Braintree.subscription_for_member(member: @current_member, id: params[:id])
+
+          begin
+            result = ::Braintree::Subscription.cancel(@subscription.subscription_id)
+            if result.success?
+              @subscription.destroy
+              render json: { success: true }
+            else
+              render json: { success: false, errors: result.errors}
```

Comment:
Yes -> https://developers.braintreepayments.com/reference/general/result-objects/ruby, it's an array, and each error has attribute, code and message.